packages/cloud_firestore/cloud_firestore/README.md

```diff
@@ -14,7 +14,73 @@ To get started with Cloud Firestore for Flutter, please [see the documentation](
+  * [Writing conditions for Security Rules](https://firebase.google.com/docs/firestore/security/rules-conditions)
+  * [Fix insecure rules](https://firebase.google.com/docs/firestore/security/insecure-rules)
+  * [Test your Security Rules](https://firebase.google.com/docs/firestore/security/test-rules-emulator)
+  * [Securely query data](https://firebase.google.com/docs/firestore/security/rules-query)
+  * [Control access per field](https://firebase.google.com/docs/firestore/security/rules-fields)
+
+* Solutions
+  * [Overview](https://firebase.google.com/docs/firestore/solutions)
+  * [Firestore Lite Web SDK](https://firebase.google.com/docs/firestore/solutions/firestore-lite)
+  * [Aggregation Queries](https://firebase.google.com/docs/firestore/solutions/aggregation)
+  * [Distributed counters](https://firebase.google.com/docs/firestore/solutions/counters)
+  * [Full-text search](https://firebase.google.com/docs/firestore/solutions/search)
+  * [Build presence](https://firebase.google.com/docs/firestore/solutions/presence)
+  * [Secure data access for users and groups](https://firebase.google.com/docs/firestore/solutions/role-based-access)
+  * [Delete data with a callable Cloud Function](https://firebase.google.com/docs/firestore/solutions/delete-collections)
+  * [Schedule data exports](https://firebase.google.com/docs/firestore/solutions/schedule-export)
+  * [Sharded timestamps](https://firebase.google.com/docs/firestore/solutions/shard-timestamp)
+  * [Geo queries](https://firebase.google.com/docs/firestore/solutions/geoqueries)
+  * [Automating database creation](https://firebase.google.com/docs/firestore/solutions/automate-database-create)
+  * [Reduce index costs with map fields](https://firebase.google.com/docs/firestore/solutions/index-map-field)
+  * [Serve bundled Firestore content from a CDN](https://firebase.google.com/docs/firestore/solutions/serve-bundles)
+
+* Usage, limits, and pricing
+  * [Usage and limits](https://firebase.google.com/docs/firestore/quotas)
+  * [Monitor usage](https://firebase.google.com/docs/firestore/monitor-usage)
+  * [Understand Cloud Firestore billing](https://firebase.google.com/docs/firestore/pricing)
+  * [Example Cloud Firestore costs](https://firebase.google.com/docs/firestore/billing-example)
+  * [Understand storage size calculations](https://firebase.google.com/docs/firestore/storage-size)
+  * [Best practices for Cloud Firestore](https://firebase.google.com/docs/firestore/best-practices)
+
+* Cloud Firestore integrations
+  * [Use the Cloud Firestore REST API](https://firebase.google.com/docs/firestore/use-rest-api)
+  * [Use Cloud Firestore and Realtime Database](https://firebase.google.com/docs/firestore/firestore-for-rtdb)
+  * [Extend with Cloud Functions](https://firebase.google.com/docs/firestore/extend-with-functions)
+  * [Third-party library integrations](https://firebase.google.com/docs/firestore/library-integrations)
```

Comment:
That's a lot of points. Don't know if that are too many points. Same as the other comments: There are many links that are not related directly to the Flutter packages but to the Firebase product of the package.